tdvt/TDVT Results.twb

```diff
@@ -59,17 +60,16 @@
       <connection class='federated'>
         <named-connections>
           <named-connection caption='test_results_combined' name='textscan.1gc4mna00d9z6u19vsqbk09t94y2leaf'>
-            <connection class='textscan' directory='/Users/tsjoblad/Desktop/new TDVT Workbook/For GitHub (1:11:21)' filename='test_results_combined.csv' password='' server='' workgroup-auth-mode='as-is' />
+            <connection class='textscan' directory='/Users/s.davis/Desktop' filename='test_results_combined.csv' password='' server='' workgroup-auth-mode='as-is' />
           </named-connection>
         </named-connections>
         <_.fcp.ObjectModelEncapsulateLegacy.false...relation connection='textscan.1gc4mna00d9z6u19vsqbk09t94y2leaf' name='test_metadata.csv' table='[test_metadata#csv]' type='table'>
-          <columns character-set='UTF-8' header='yes' locale='en_US' separator='&#9;'>
-            <column datatype='string' name='Test Function' ordinal='0' />
-            <column datatype='string' name='Test Category' ordinal='1' />
-            <column datatype='string' name='Test Path Categorized' ordinal='2' />
-            <column datatype='string' name='Test Name Categorized' ordinal='3' />
-            <column datatype='string' name='Function Categorized' ordinal='4' />
-            <column datatype='string' name='Priority' ordinal='5' />
+          <columns character-set='UTF-8' header='yes' locale='en_US' separator=','>

```

Comment:
`separator` should still be tab

Reply:
Confusing. test_results_combined.csv, which is what this workbook uses, it comma separated, and if the WB is saved with a tab expectation, the relationship is unreadable and the workbook won't work, showing errors.

Reply:
test_results_combined.csv is comma separated. test_metadata.csv is tab separated.

Reply:
<img width="1131" alt="2023-07-19_15-02-07" src="https://github.com/tableau/connector-plugin-sdk/assets/5643640/dfd682a5-24d6-4636-9bcc-c19bac09a735">
@tabsdavis try changing this, saving the workbook, and doing a git diff.

Reply:
My local copy of test_metadata.csv somehow got saved with a comma format. (Maybe by the Numbers app?) I verified the repo contains the tab delimiter so you are corrected. Redownloaded CSV and tested. Committed the change.